golang/Dockerfile.test

```diff
@@ -0,0 +1,8 @@
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:alpine AS builder
+
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+RUN printf "I am running on ${BUILDPLATFORM:-linux/amd64}, building for ${TARGETPLATFORM:-linux/amd64}\n$(uname -a)\n"
+
+COPY wrapper.sh /go/bin/go
+RUN go env
```

Comment:
Instead of separate file can you add it as a test stage

Reply:
Yeah, you're right. I merged it in the Dockerfile.